docassemble/ALToolbox/display_template.py

```diff
@@ -14,10 +21,18 @@ def display_template(
         adjust_height = (
             f"onmouseover=\"this.style.height = (this.scrollHeight) + 'px';\""
         )
-
+        
+    # Introducing `classname` to try to align with `collapse_template`
+    if not class_name:
+      class_name = classname
     class_name = class_name.strip()
+    
+    container_classname_plus = "al_display_template"
+    if container_classname:
+      container_classname_plus += f" { container_classname }"
 
     the_id = re.sub(r"[^A-Za-z0-9]", "", template.instanceName)
+    actual_container_id_tag = container_id_tag or f"{ the_id }_container"
```

Comment:
What's the movitation for allowing people to set the container id directly, or why can't it just always be `{ the_id }_container`?

Reply:
Because that way they can use the id in ALKiln to detect elements (in a future feature) without worrying about changing their variable's instance name.

Reply:
See [lower down](https://github.com/SuffolkLITLab/docassemble-ALToolbox/pull/174#issuecomment-1515161647) for more responses to this.